packages/client/src/v2-events/hooks/usePrintableCertificate.ts

```diff
@@ -127,6 +136,18 @@ export const usePrintableCertificate = ({
         certificateConfig?.id
   ).length
 
+  // Offline flow
+  const { getLocalDraftOrDefault, getRemoteDraftByEventId } = useDrafts()
+  const remoteDraft = getRemoteDraftByEventId(event.id)
+  const eventDrafts = remoteDraft
+    ? getEventDrafts(event.id, getLocalDraftOrDefault(remoteDraft), [
+        remoteDraft
+      ])
+    : []
+  const localDeclaration = eventDrafts[0]?.action?.declaration as
```

Comment:
Could we get rid of the casting?

Reply:
If I remove this casting here, I am unable to print the declaration offline,
I am getting a conversion error